MOxUnit/moxunit_runtests.m

```diff
@@ -43,7 +43,7 @@
 % NNO Jan 2014
 
 
-    [verbosity, filenames, fid]=get_params(varargin{:});
+    [verbosity, filenames, fid, junitxml]=get_params(varargin{:});
```

Comment:
As mentioned earlier: how about adding a 'output_format' argument which can be set to text or xml?